src/AccessibilityInsights.sln

```diff
@@ -96,6 +96,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MsiFileTests", "MsiFileTest
 		{137DB774-6464-48BC-B577-091ABEAF0178} = {137DB774-6464-48BC-B577-091ABEAF0178}
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{93F62CF7-9A3E-4402-BB45-67F66082EAFA}"
```

Comment:
Just one question for my own information: what does the ID between the curly brackets mean?

Reply:
@ThanyaLeif, I admit ignorance here. I added this via the IDE and this is what it changed.

Reply:
I see, no problem.
It looks good overall so I'll approve it.